pkg/controller/fileobserver/observer_polling.go

```diff
@@ -19,6 +19,11 @@ type pollingObserver struct {
 	reactors map[string][]reactorFn
 	files    map[string]string
 
+	// If files are newer then startTimestamp the observer will report a change for them.
+	// This exists to mitigate issue when we start observing files but have to synchronize
+	// the observing with the process consuming the files.
+	startTimestamp *time.Time
```

Comment:
you can avoid the pointer if you use the zero value instead. Isn't there `time.IsZero`?

Reply:
there is no `IsZero`, i was thinking about that as well, but have not find a nice way to do this.